docs/getting-started/install-endpoint.asciidoc

```diff
@@ -0,0 +1,72 @@
+
+NOTE: Configuring the Endpoint Integration on the Elastic Agent requires that the user have permission to use Ingest Manager in Kibana.
+
+[float]
+== Before you begin
+
+If you're using the Elastic Agent on macOS Mojave (10.14) or later, ensure that you have enabled <<sensor-full-disk-access,Full Disk Access>>. Lastly, review the Elastic Security system requirements.
+
+[float]
+== Add Elastic Security integration
+
+1. In Kibana, select **Security** > **Administration**. If this is not your first time using Elastic Security, select **Ingest Manager** > **Integrations** and search for **Elastic Endpoint Security**.
+2. On the Administration page of the security app or the Elastic Endpoint Security integration page in Ingest Manager, select **Add Endpoint Security**. The integration configuration page appears.
++
+[role="screenshot"]
+image::images/install-endpoint/security-integration.png[]
++
+3. Select a configuration for the Elastic Agent. You can use either the **Default config**, which works on most hosts, or adds security integration to a custom or existing configuration. For more details on Elastic Agent configuration settings, see {ingest-guide}/elastic-agent-configuration.html[Configuration settings].
+4. Configure the Elastic Endpoint Security integration with a name and options description. When done configuring, select **Save integration**. Kibana redirects you back to the administration section of the security app.
++
+[role="screenshot"]
+image::images/install-endpoint/add-elastic-endpoint-security.png[]
++
+5. On the Enable Elastic Endpoint Security on your Agent's page, select the name of your new integration. To enroll your Agents with Endpoint Security, select **Enroll Agent**.
+6. Kibana redirects you back to Ingest manager to add the Elastic Agent to your host.
+
+[float]
+== Configure and enroll Elastic Agent
+
+There are two methods to install the Elastic Agent with the Elastic Endpoint Security integration using Ingest Manager.
+
+=== Enroll with Fleet
+
+1. Go to Ingest Manager. Select **Overview** > **Add agent**.
++
+[role="screenshot"]
+image::images/install-endpoint/add-agent.png[]
++
+2. In the Add agent pane of the Configurations section, download the Elastic Agent on your host's machine.
+3. After the download is complete, select an agent configuration. The selected configurations should include both **System** and **Elastic Endpoint Security** and any other integrations you chose to include when configuring an integration.
++
+[role="screenshot"]
+image::images/install-endpoint/endpoint-configuration.png[]
++
+4. After the Elastic Agent is installed on your host machine, open a command-line interface, and navigate to your Agent's directory. Copy the commands from Ingest Manager for your OS to enroll and run the Agent.
```

Comment:
this looks nice to me, glad we aren't trying to repeat steps (see note below for Stand-alone mode, maybe we shouldn't cite any specific 'command' steps there either...  and instead here, and there, we can link to the Ingest Manager docs page for installing Agent.  Thoughts?

Reply:
Having said that I see nothing bad about the above text, its great - I just think a link would be great too.  :)

Reply:
@EricDavisX: Decided to add the OS commands underneath the image, like they appear in the the Ingest Manager docs. Let me know if that works.

Reply:
what ever you think is best - sounds good to me.